tests/mocks/section1mSimpleQueryMock.ts

```diff
@@ -0,0 +1,9 @@
+export default `section Section1;
+
+        shared Query1 =
+
+    let
+        Source = Folder.Files("C:\\Users\\user1\\Desktop\\test")
+    in
+        Source
+    ;`;
```

Comment:
Maybe have the 2 mocks at the same file (since its unlikely they would be used separately) and also have the mock of section1m take its query part from the simpleQuery mock (a bit more future proof)?